src/site/_data/site.js

```diff
@@ -21,4 +21,5 @@ module.exports = {
   url: 'https://web.dev',
   repo: 'https://github.com/GoogleChrome/web.dev',
   subscribe: 'https://web.dev/subscribe',
+  imageCdn: 'https://webdev.imgix.net',
```

Comment:
Is it worth adding a comment to this? It's something in our main config which references a third-party site. We'll be adding more of these, but it seems like lock-in for someone wanting to run our code themselves.

Reply:
yeah I can add a comment. This should only kick in if the person is doing a production build.